src/pages/Channel.js

```diff
@@ -129,6 +129,12 @@ function Channel(props) {
     </span>;
   }
 
+  let lastMessage = channel.messages[channel.messages.length - 1];
```

Comment:
This is nice, but I think UX-wise people would expect to get their last message here. It doesn't have to come from the channel history, though.

The way I imagine it (and I think the way it works in most IRC clients) is that there is a short array of the messages (say last 10 messages) that **you** sent on the channel, and you can change between them with up and down arrows.